pkg/operator/vtgate/deployment.go

```diff
@@ -95,6 +95,7 @@ type Spec struct {
 	ExtraEnv          []corev1.EnvVar
 	ExtraVolumes      []corev1.Volume
 	ExtraVolumeMounts []corev1.VolumeMount
+	Annotations       map[string]string
```

Comment:
Remove the old `PodAnnotations` field to make sure we aren't accidentally using it.

Reply:
Done.